options/locale/locale_en-US.ini

```diff
@@ -922,6 +922,16 @@ settings.protected_branch=Branch Protection
 settings.protected_branch_can_push=Allow push?
 settings.protected_branch_can_push_yes=You can push
 settings.protected_branch_can_push_no=You can not push
+settings.branch_protection = Branch Protection for <b>%s</b>
+settings.protect_this_branch = Protect this branch
+settings.protect_this_branch_desc = Disable force pushes and prevent from deletion.
```

Comment:
I would suggest `Disable force pushes and prevent deletion.`

Reply:
done.